R/pkg/inst/tests/testthat/test_Windows.R

```diff
@@ -20,7 +20,7 @@ test_that("sparkJars tag in SparkContext", {
   if (.Platform$OS.type != "windows") {
     skip("This test is only for Windows, skipped")
   }
-  testOutput <- launchScript("ECHO", "a/b/c", capture = TRUE)
+  testOutput <- launchScript("ECHO", "a/b/c", wait = TRUE)
```

Comment:
Hmm, I've tried, I don't think it would work.
When calling `system2(.., wait = FALSE, capture = "")` the output to stdout is actually from the child process, so I don't think we would be able to see it from the R process.
We could redirect it, but then it would be the same as `system2(..., wait = FALSE, capture = TRUE)` but again it wouldn't be what we are normally calling.

I think we would need to dig deeper on this.